keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -984,26 +1008,31 @@ First, there will be no effect on any workload that doesn't use a new field. Any
 combination of feature gate enabled/disabled or version skew will work as usual
 for that workload.
 
-So when the new functionality wasn't yet used, downgrade will not be affected.
+When the new functionality wasn't yet used, downgrade will not be affected.
 
 Due to the new field added to `initContainers` to turn them into sidecars,
 downgrading to the version without this feature will make all Pods using this
-flag unscheduleable. New Pods will be rejected by the control plane and
-all kubelets. Pods that has already been created will not be rejected by control
-plane, but once reaching the kubelet, that has this feature disabled or which
-is old, kubelet will reject the Pod on unmarshalling.
-
-**Note**, we tested kubelet behavior. For the control plane we may need
-to implement a new logic to reject such Pods when feature gate got turned off.
+flag unscheduleable. New Pods will be rejected by the control plane.
+
+Pods that has already been created will not be rejected by control
+plane nor by kubelet. Both will treat the sidecar container as an Init container.
+This may render the Pod unusable as it will stuck in initialization forever -
+sidecar container are never exiting. We will document this behavior for Alpha release.
+Promoting feature to Beta we will revisit the situation. If we will see this as
+a major issue, we will need to wait for 3 releases so kubelet will have the logic
```

Comment:
Does k8s generally support downgrade from a version with an alpha feature enabled?

Reply:
For alpha it is not that critical. For Beta we will need to review this